### DISCUSSION TITLE
More API / Events

### DISCUSSION THREAD

**Discussion by Pikaronga** [BG-Software-LLC/SuperiorSkyblock2#428] [Denied Feature Requests] [answered]
Add a api for developers to hook into when deleting a island to get all the members that where in that island so we can execute things to happen when a island is deleted same for when a player is kicked or when a player leaves a island. (This would greatly help developers be able to remove stuff from the island members when they are no longer in the island).

**Comment by OmerBenGera** [comment, marked as answer]
Island#getAllPlayersInside to get a list of all players in the area of the island, or Island#getIslandMembers for a list of all the members of the island.
Also, events exist for all the listed actions.
